docs/relevancyimplementations.rst

```diff
@@ -0,0 +1,20 @@
+Relevancy Implementations
+-------------------------
+
+RankRelevancyWithMarginSampledFeedback
+++++++++++++++++++++++++++++++++++++++
+
+.. autoclass:: smqtk_relevancy.impls.margin_sampling.RankRelevancyWithMarginSampledFeedback
```

Comment:
This and the other two autodocs are failing noting that the module path provided does not resolve. I think you are missing a module after `impls`.